src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/Row.js

```diff
@@ -140,6 +204,15 @@ export default class Row extends React.PureComponent<Props> {
         });
     };
 
+    handleToggleChange = () => {
+        const {id, rowIndex, expanded} = this.props;
+        const identifier = id || rowIndex;
+
+        if (this.props.onToggleChange && identifier) {
+            this.props.onToggleChange(identifier, !expanded);
```

Comment:
Now I see what this callback is for :see_no_evil: I think `toggle` is a little bit an unlucky way, because you can toggle any flag. Maybe we should replace it by two callbacks: `onExpand` and `onCollapse`? Then it would be clearer what it is doing.

Reply:
@danrot sound good for me.